front/src/modules/simulationResult/components/SpeedSpaceChart/consts.ts

```diff
@@ -24,3 +24,18 @@ export const electricalProfileColorsWithoutProfile = {
   '15000V': '#009AA6',
   '3000V': '#1FBE00',
 };
+
+export const electricalProfilesDesignValues = {
```

Comment:
it's already there for the GEV V1, if we don't put it this way I really don't know how do!

for the G, the mockup doesn't mention it but you're right it can exist (as many others that are not specified from the beginning), I'll set it

Reply:
done